dotcom-rendering/tsconfig.json

```diff
@@ -47,6 +47,9 @@
             /* Aliases should also be added to the webpack, babel and jest configurations */
             "*": ["node_modules/@types/*", "*"], // Make sure that yarn linking doesn't confuse things https://github.com/microsoft/TypeScript/issues/11916#issuecomment-257130001
         },
+
+        "skipLibCheck": true,
```

Comment:
is this ok? this make TS more permissive "the expense of type-system accuracy", as the docs say. do we want that?

Reply:
I guess I’d like not to…